manager/assets/modext/sections/security/profile/update.js

```diff
@@ -182,6 +320,7 @@ Ext.extend(MODx.panel.UpdateProfile,MODx.FormPanel,{
             ,listeners: {
                 'success': {fn:function(r) {
                     this.getForm().setValues(r.object);
+                    Ext.get('modx-profile-header').update(_('profile')+': '+r.object.username);
```

Comment:
Please wrap `r.object.username` in a htmlencode (from memory: `Ext.util.Format.htmlEncode(r.object.username)`)  to make sure this doesn't introduce a XSS vulnerability.

Reply:
Done. Check please